datus-spark/tests/integration/conftest.py

```
@@ -0,0 +1,197 @@
+
+
+@pytest.fixture
+def connector(config: SparkConfig) -> Generator[SparkConnector, None, None]:
+    """Create and cleanup Spark connector for integration tests."""
+    conn = None
+    try:
+        conn = SparkConnector(config)
+        if not conn.test_connection():
+            pytest.skip("Database connection test failed")
+        yield conn
+    except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)